js/healthmap.js

```diff
@@ -265,10 +266,12 @@ function processHash(url) {
         document.body.classList.add('autodrive');
         continue;
       }
-      if (hashBrown.toLowerCase() == '3d') {
```

Comment:
Oh, actually, I had looked at this a bit too quickly, but you're removing the 3d feature here -- is that on purpose?

Reply:
No - totally an accident. Sorry!